ts/features/bonus/bpd/saga/networking/ranking.ts

```diff
@@ -43,6 +52,9 @@ export function* bpdLoadRaking(
       );
     }
   } catch (e) {
+    void mixpanelTrack(mixpanelActionFailure, {
+      reason: e.message
```

Comment:
```suggestion
      reason: getError(e).message
```
`e` could be a string (max-retries)